common/templates/license.html

```diff
@@ -0,0 +1,55 @@
+<%page args="license, button=False, button_size='88x31'"/>
+## keep this synchronized with the contents of cms/templates/js/license-selector.underscore
+<%!
+from django.utils.translation import ugettext as _
+
+def parse_license(lic):
+    """
+    Returns a two-tuple: license type, and options.
+    """
+    if not lic:
+        return None, {}
+    if ":" not in lic:
+        # no options, so the entire thing is the license type
+        return lic, {}
+
+    ltype, option_str = lic.split(":", 1)
+    options = {}
+    for part in option_str.split():
+        if "=" in part:
+            key, value = part.split("=", 1)
+            options[key] = value
+        else:
+            options[part] = True
+    return ltype, options
+%>
+<% license_type, license_options = parse_license(license) %>
+% if license_type == "all-rights-reserved":
+    © <span class="license-text">${_("All Rights Reserved")}</span>
```

Comment:
@cptvitamin I'm unfamiliar with microdata. Can you elaborate? What's the advantage? Is it something we could add in later?